src/components/documentation/edit-tabs/edit-page-content.components.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IScope } from 'angular';
+import '@gravitee/ui-components/wc/gv-switch';
+import { PageType } from '../../../services/documentation.service';
+
+interface IPageScope extends IScope {
+  fetcherJsonSchema: string;
+  rename: boolean;
+  editorReadonly: boolean;
+  currentTab: string;
+  currentTranslation: any;
+  acls: any;
+}
+
+const EditPageAclsComponent: ng.IComponentOptions = {
```

Comment:
We can `export const EditPageAclsComponent` directly and use `import { EditPageAclsComponent } ...` in management module instead of using a default export https://github.com/gravitee-io/gravitee-management-webui/pull/1611/files#diff-5321c21486355a1b6b9b6ac27cb64f24c3d1de759580116554623c4857906954R62